hoomd/md/pair/aniso.py

```diff
@@ -248,3 +251,206 @@ def type_shapes(self):
             A list of dictionaries, one for each particle type in the system.
         """
         return super()._return_type_shapes()
+
+
+class ALJ(AnisotropicPair):
+    r"""Anistropic LJ potential.
+
+    Args:
+        r_cut (float): Default cutoff radius (in distance units).
+        nlist (:py:mod:`hoomd.md.nlist`): Neighbor list
+        name (str): Name of the force instance.
+        average_simplices (bool): Whether or not to perform simplex averaging
+            (see below for more details).
+
+    :py:class:`alj` computes the LJ potential between anisotropic particles.
```

Comment:
```suggestion
    :py:class:`ALJ` computes the Lennard-Jones potential between anisotropic particles.
```
Also, add a reference to the methods paper.